kinto/tests/test_configuration/test.ini

```diff
@@ -1,3 +1,7 @@
+# Created at config_file_timestamp
```

Comment:
Why do you have this `config_file_timestamp` string here ? I thought the file content was compared somewhere else, no ?